packages/libxext.rb

```diff
@@ -1,32 +1,29 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Libxext < Package
+class Libxext < Autotools
   description 'library for the X window system'
   homepage 'https://x.org'
-  version '1.3.4'
+  version '1.3.6'
   license 'MIT'
-  compatibility 'all'
-  source_url 'https://www.x.org/releases/individual/lib/libXext-1.3.4.tar.gz'
-  source_sha256 '8ef0789f282826661ff40a8eef22430378516ac580167da35cc948be9041aac1'
-  binary_compression 'tar.xz'
+  compatibility 'x86_64 aarch64 armv7l'
+  source_url 'https://www.x.org/releases/individual/lib/libXext-1.3.6.tar.xz'
+  source_sha256 'edb59fa23994e405fdc5b400afdf5820ae6160b94f35e3dc3da4457a16e89753'
+  binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '71402bff06db86bd9b08030c7615868a8bfcf3ed52ef9369d066267c1827c118',
-     armv7l: '71402bff06db86bd9b08030c7615868a8bfcf3ed52ef9369d066267c1827c118',
-       i686: '1aee8625004cb1872da572c6ca962717d421c8245e2c79c418425c86babd1cab',
-     x86_64: '9024bf186472eeb3ed2fe0ea7c77716f76c9860248418d8f804f600d5b5c2704'
+    aarch64: '2f74453c3b6d4dee2cd194c59134e0fd14f1dc3fcd240edfbd2976021de0c27d',
+     armv7l: '2f74453c3b6d4dee2cd194c59134e0fd14f1dc3fcd240edfbd2976021de0c27d',
+     x86_64: '99bd31ebf5bf87c2587ac10bcafc7719a70629660dfb366b0362604a9228d6a1'
   })
 
+  depends_on 'glibc' # R
+  depends_on 'libx11' # R
+  depends_on 'libxau' # R
+  depends_on 'libxcb' # R
+  depends_on 'libxdmcp' # R
   depends_on 'llvm16_lib' => :build
```

Comment:
Oops! This should be updated too to `llvm18_lib`

Reply:
That would require new binaries, right?

Reply:
If getrealdeps.rb doesn't complain, then no rebuild is necessary.

Reply:
Rebuilt binaries anyway to make sure it works.